indico/web/rh.py

```diff
@@ -408,3 +408,14 @@ def json_errors(rh):
     """
     rh._JSON_ERRORS = True
     return rh
+
+
+def cors_enabled(f):
+    """Enable CORS for this endpoint."""
+    @wraps(f)
+    def wrapper(*args, **kwargs):
+        response = f(*args, **kwargs)
+        response = current_app.make_response(response)
```

Comment:
I don't think it's a good idea to convert to a response here...  maybe we can instead set something on the RH instance and do it in the RH code after calling `response = current_app.make_response(res)` (which is already there)?

Reply:
yeah that sounds like a better option